src/libraries/Common/tests/System/IO/Compression/LocalMemoryStream.cs

```diff
@@ -27,6 +27,7 @@ public static async Task<LocalMemoryStream> readAppFileAsync(string testFile)
         return ms;
     }
 
+    [ActiveIssue("https://github.com/dotnet/runtime/issues/36845", TestPlatforms.Android)]
```

Comment:
This should be removed because it only applies to tests.

Reply:
Added a fix to my Kusto query to avoid this false positive